### ISSUE TITLE
is there gonna be a version that support 32 bits in the future?

### ISSUE THREAD

**Issue by hiyoiko** [EKA2L1/EKA2L1#339]
hello. i've recently start to try the EKA2L1 (thanks to a video on Youtube) to play a specific N-Gage 2.0 game; Resident Evil Degeneration and I have it for my Xiaomi Redmi 5A. When I run it for my "outdated" android phone, it is slow to respond, sometime it freezes and when finally able to get it to work and play the game, it lags. When I asked about why it lags for me, but not in the video i watched, the person said "you need a 64 bits phones" so, the conclusion is there and all i have is a 32 bits phone, it will still work, but i have to deal with the said issues due to my own fault for not have a proper device/os... same for the windows version, which require a 64 bits Windows, some stuffs like an OpenGL 3.1, etc. So, the question: is there gonna be a version that would support a 32 bits or not? I highly doubt there would be, but no harm for me to ask. 

thank you for the software you've made and good luck in the future for you & the dev team

**Comment by pent0** [comment]
We are supporting 32-bit phone, the person probably misunderstood. We wrote our own CPU recompiler for 32-bit, however it is not performed as fast as 64-bit. If it's for performance, I highly suggested to upgrade to 64-bit.

See #235 , resolved some months ago. The game seems to perform fine on Samsung A10 though, but your phone is a bit ancient.

**Comment by hiyoiko** [comment]
> We are supporting 32-bit phone, the person probably misunderstood. We wrote our own CPU recompiler for 32-bit, however it is not performed as fast as 64-bit. If it's for performance, I highly suggested to upgrade to 64-bit.
> 
> See #235 , resolved some months ago. The game seems to perform fine on Samsung A10 though, but your phone is a bit ancient.

That i don't have a doubt about performance difference between 32 bits and 64 bits phones to run this emulator, learnt it the hard way and the fact my phone is an ancient leftover. Ohhh So there is already a 32 bits version for android and it's experimental. I still curious before closing the case. About the windows version, is it only for 64 bits?

**Comment by pent0** [comment]
Yes, if there's no 32-bit version, you can not even launch the app at all.

And yes, the Windows version is only 64-bits, we use Dynarmic CPU recompiler that is used in Citra Yuzu, and they only support ARM64 and x86_64. We don't plan on a 32-bit version PC.
